x-pack/legacy/plugins/siem/server/kibana.index.ts

```diff
@@ -39,7 +35,7 @@ export const initServerWithKibana = (
   }
   kbnServer.injectUiAppVars('siem', async () => kbnServer.getInjectedUiAppVars('kibana'));
 
-  const libs = compose(kbnServer, mode);
+  const libs = compose(core, kbnServer);
   initServer(libs);
   if (
     kbnServer.config().has('xpack.actions.enabled') &&
```

Comment:
@rudolf will the Detection Engine routes' dependence on the alerting client (via `request.getAlertsClient()`) prevent them from being converted to the new router? I don't see a way to retrieve an alerting client without a legacy request so I assume that means "yes," but please do verify that assumption.

Reply:
Followup: I posed this question to the platform team. While there may be a way to leverage `AlertsClientFactory` and the internal, protected `Hapi.Request` to get a legacy client in a new route, the consensus was that that's probably not a worthwhile endeavor. Going to leave the Detection Engine routes until the new alerting plugin is ready.